ci-operator/step-registry/cucushift/hypershift-extended/enable-mgmt/cucushift-hypershift-extended-enable-mgmt-commands.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [ -f "${SHARED_DIR}/cluster-type" ] ; then
+    CLUSTER_TYPE=$(cat "${SHARED_DIR}/cluster-type")
+    if [[ "$CLUSTER_TYPE" == "osd" ]] || [[ "$CLUSTER_TYPE" == "rosa" ]]; then
+        cp "${SHARED_DIR}/kubeconfig" "${SHARED_DIR}/nested_kubeconfig"
+        cat "${SHARED_DIR}/hs-mc.kubeconfig" > "${SHARED_DIR}/kubeconfig"
+    fi
+fi
```

Comment:
if not mgmt cluster or rosa ,  we should exit -1 right ? If not,  the workerflow will continue go to next steps.
if [ -f "${SHARED_DIR}/cluster-type" ] ; then 
    if [[ "$CLUSTER_TYPE" == "osd" ]] || [[ "$CLUSTER_TYPE" == "rosa" ]]; then
    else 
          exit -1
    fi
else
      exit -1
fi